skyvern-frontend/src/components/ThemeProvider.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from "react";
+import { Theme, ThemeProviderContext } from "./themeProviderContext";
+
+export type ThemeProviderProps = {
+  children: React.ReactNode;
+  defaultTheme?: Theme;
+  storageKey?: string;
+};
+
+export function ThemeProvider({
```

Comment:
is this ThemeProvider generated?
I would love to see the UI screenshot if any!

Reply:
It's from [here](https://ui.shadcn.com/docs/dark-mode/vite) but I didn't add the toggle for it, I just wanted to have dark by default like the streamlit app currently has.